concepts/randomness/about.md

```diff
@@ -0,0 +1,93 @@
+# Randomness Introduction
+
+Package [`math\rand`][mathrand] provides support for generating pseudo-random numbers.
+
+Here is how to generate a random integer between `0` and `99`:
+
+```go
+n := rand.Intn(100) // n is a random int, 0 <= n < 100
+```
+
+Function `rand.Float64` returns a random floating point number between `0.0` and `1.0`:
+
+```go
+f := rand.Float64() // f is a random float64, 0.0 <= f < 1.0
+```
+
+There is also support for shuffling a slice (or other data structures):
+
+```go
+x := []string{"a", "b", "c", "d", "e"}
+// shuffling the slice put its elements into a random order
+rand.Shuffle(len(x), func(i, j int) {
+	x[i], x[j] = x[j], x[i]
+})
+```
+
+## Seeds
+
+The number sequences generated by package `math\rand` are not truly random.  
+They are completely determined by an initial `seed` value.
+By default, the package uses the number `1` as a seed.
+
+In order to get a different sequence of random numbers, we need to set a different seed.
+A simple approach is to seed with the current computer time.
+
+```go
+rand.Seed(time.Now().UnixNano())
+```
+
+Seeding must be done **before** any of the functions that generate random numbers.
```

Comment:
```suggestion
Setting the seed must be done **before** calling any of the functions that generate random numbers. We usually only need to set the seed once in our program. From one seed we can generate many random numbers.
```

Additional clarification. Also apply this change in the other places where this is present.

Reply:
Done :)